backend/src/main/kotlin/com/ksidelta/libruch/modules/borrowing/BorrowingReadModel.kt

```diff
@@ -0,0 +1,40 @@
+package com.ksidelta.libruch.modules.borrowing;
+
+import com.ksidelta.libruch.modules.kernel.Party
+import org.axonframework.eventhandling.EventHandler
+import org.axonframework.messaging.MetaData
+import org.springframework.data.repository.CrudRepository
+import org.springframework.http.codec.multipart.Part
+import org.springframework.stereotype.Repository
+import org.springframework.stereotype.Service
+import java.util.*
+import javax.persistence.Entity
+import javax.persistence.Id
+
+@Service
+class EventProcessor(val borrowingReadModelRepository: BorrowingReadModelRepository) {
+    @EventHandler
+    fun handle(event: RegisterNewBorrowing, metadata: MetaData) =
```

Comment:
You should use this creation event here, not the command name

Reply:
ok